driver/src/test/java/org/neo4j/driver/internal/InternalSessionTest.java

```diff
@@ -78,4 +78,34 @@ public void shouldBeAbleToOpenTxAfterPreviousIsClosed() throws Throwable
         // Then we should've gotten a transaction object back
         assertNotNull( tx );
     }
+
+    @Test
+    public void shouldNotBeAbleToUseSessionWhileOngoingTransaction() throws Throwable
```

Comment:
These two tests has nothing to do with the fix, I just thought they were missing.